towhee/functional/mixins/milvus_mixin.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from towhee.hparam import param_scope
+from ..entity import Entity
+
+
+def insert(self, index, collection, batch_size):
+    # pylint: disable=import-outside-toplevel
+    from towhee.utils.milvus_utils import Collection, MutationResult
+    if isinstance(collection, str):
+        collection = Collection(collection)
+    primary_keys = []
+    insert_count = 0
+    data = [[] for _ in range(len(index))]
+
+    for entity in self:
+        for i in range(len(index)):
+            data[i].append(getattr(entity, index[i]))
+        if len(data[0]) == batch_size:
+            mr = collection.insert(data)
+            data = [[] for _ in range(len(index))]
+            primary_keys += mr.primary_keys
+            insert_count += mr.insert_count
+            print(f'Successfully inserted {mr.insert_count} row data.')
+
+    if len(data[0]) > 0:
+        mr = collection.insert(data)
+        primary_keys += mr.primary_keys
+        insert_count += mr.insert_count
+        print(f'Successfully inserted {mr.insert_count} row data.')
+
+    e = Entity(insert_count=insert_count,
+               primary_keys=primary_keys,
+               delete_count=mr.delete_count,
+               upsert_count=mr.upsert_count,
+               timestamp=mr.timestamp)
+    milvus_mr = MutationResult(e)
+    collection.load()
+    return milvus_mr
+
+
+def _to_milvus_callback(self):
+    # pylint: disable=consider-using-get
+    def wrapper(_: str, index, *arg, **kws):
+        batch_size = 1
+        if isinstance(index, str):
+            index = (index,)
+
+        if arg is not None and len(arg) == 1:
+            collection, = arg
+        elif arg is not None and len(arg) == 2:
+            collection, batch_size = arg
+
+        if 'collection' in kws:
+            collection = kws['collection']
+        if 'batch' in kws:
+            batch_size = kws['batch']
+
+        milvus_mr = insert(self, index, collection, batch_size)
+        return milvus_mr
+    return wrapper
+
+
+class MilvusMixin:
+    """
+    Mixin about Milvus, such as load data to Milvus collection.
+    """
+    def __init__(self):
+        super().__init__()
+        self.to_milvus = param_scope().callholder(_to_milvus_callback(self))
```

Comment:
will sphinx generate right document for `dc.to_milvus`? can you make a try with `python setup.py build_sphinx`

Reply:
No, I haven't added `MilvusMixin` to *mixins.rst* and will add it later.

And another question, there is no example for `to_milvus`, how about installing milvus in CI and add doctest for it.

Reply:
> No, I haven't added `MilvusMixin` to _mixins.rst_ and will add it later.
> 
> And another question, there is no example for `to_milvus`, how about installing milvus in CI and add doctest for it.

I think we should avoid heavy operations in the CI workflow.